src/actions/speaker-actions.js

```diff
@@ -148,7 +206,8 @@ export const getSpeakers =
     const params = {
       page,
       per_page: perPage,
-      access_token: accessToken
+      access_token: accessToken,
+      fields: "id,first_name,last_name,email,member_id"
```

Comment:
i do think that u could reduce the time to the half if u add relations:"none"
currently is still retrieving a lot of not used data